tensilelite/rocisa/cmake/RocisaConfig.cmake

```diff
@@ -0,0 +1,74 @@
+################################################################################
+#
+# Copyright (C) 2022-2025 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+################################################################################
+
+include(CMakeParseArguments)
+if(NOT DEFINED Rocisa_ROOT)
+    # Compute the installation prefix relative to this file.
+    get_filename_component(Rocisa_PREFIX "${CMAKE_CURRENT_LIST_FILE}" PATH)
+    get_filename_component(Rocisa_PREFIX "${Rocisa_PREFIX}" PATH)
+    if (WIN32)
+        execute_process(COMMAND "${Rocisa_PREFIX}/bin/RocisaGetPath.exe" OUTPUT_VARIABLE Rocisa_ROOT)
+    else()
+        execute_process(COMMAND "${Rocisa_PREFIX}/bin/RocisaGetPath" OUTPUT_VARIABLE Rocisa_ROOT)
+    endif()
+endif()
+
+message(STATUS "Building rocIsa" ${Rocisa_ROOT})
+
+set(ROCISA_VIRTUAL_DIR ${Rocisa_ROOT})
+file(MAKE_DIRECTORY ${ROCISA_VIRTUAL_DIR}/build)
+execute_process(
+  COMMAND ${CMAKE_COMMAND} -DPython_EXECUTABLE=${VIRTUALENV_BIN_DIR}/${VIRTUALENV_PYTHON_EXENAME} ..
+  WORKING_DIRECTORY ${ROCISA_VIRTUAL_DIR}/build
```

Comment:
I'm having a bit of trouble following the paths, but this looks like you are creating the `build` directory under the source dir if I am following. Can you please not do that? If you need a place to stash it, at least use something like `${PROJECT_BINARY_DIR}/rocisa_build` (or something -- so long as it is explicitly in the build tree).

Reply:
Fixed